src/components/EventsList.tsx

```diff
@@ -18,11 +17,7 @@ const getDateText = (event: Event) => {
   return `${day} at ${event.startTime} - ${event.endTime}`;
 };
 
-const EventsList = ({ events, count }: EventsListProps) => {
-  events = events.slice(0, count);
-  events = events.sort((a, b) => {
```

Comment:
any reason why you removed the sort? this line basically ensures that the events data is rendered in order, so we don't have to strictly maintain one in parent

Reply:
Oh yeah so the idea here was to have past events sorted in reverse order, since before it was sorting it so that the oldest event came up first in pastEvents. But depends on how we want it to display events, we could either have it so that it's reverse-chronological order for all events or make it so that upcoming events is sorted chronologically while past events is sorted in reverse-chonological order.

<img width="877" alt="Screenshot 2023-11-10 at 10 42 41 PM" src="https://github.com/BUGS-NYU/bugs-nyu.github.io/assets/116298054/f5862c8b-7358-4de6-91d2-6f931fc22ba8">
This is what it looks like now

Reply:
So I'm sorting `upcomingEvents` and `pastEvents` separately. Here is some context on what it used to look like:
<img width="836" alt="Screenshot 2023-11-10 at 10 46 50 PM" src="https://github.com/BUGS-NYU/bugs-nyu.github.io/assets/116298054/9b91fe80-14ef-4f30-90dc-0e5ba9b4e54a">

Reply:
yeah the way you did it makes much more sense